examples/pipeline_wavernn/wavernn.py

```diff
@@ -0,0 +1,586 @@
+import argparse
+import logging
+import os
+import signal
+from collections import defaultdict
+from datetime import datetime
+from time import time
+
+import torch
+import torch.nn as nn
+import torchaudio
+from transform import Transform
+from datasets import datasets_ljspeech, collate_factory
+from typing import List
+from torchaudio.models import _WaveRNN
+from torch.utils.data import DataLoader
+from torchaudio.datasets.utils import bg_iterator
+from torch.optim.lr_scheduler import ExponentialLR, ReduceLROnPlateau
+from torch.optim import Adam
+from mol_loss import Mol_Loss
+from utils import MetricLogger, count_parameters, save_checkpoint
+
+SIGNAL_RECEIVED = False
+
+
+def parse_args():
```

Comment:
Are the defaults your best ones so far?

Reply:
Yes, now the arguments are the best for training.